### PR TITLE
Added/updated dutch translations

### PR DESCRIPTION
**Pull request by bartee**
Updated main translations, as well as created the appropriate translations for plugins.

### CODE REVIEW

File: plugins/talk-plugin-ignore-user/client/translations.yml [hunk 1]
```diff
@@ -62,17 +62,17 @@ fr:
     confirmation_title: Ignore {0}?
 nl_NL:
   talk-plugin-ignore-user:
-    section_title: Ignored users
-    section_info: Because you ignored the following commenters, their comments are hidden.
-    stop_ignoring: Stop ignoring
-    ignore_user: Ignore User
-    cancel: Cancel
+    section_title: Genegeerde gebruikers
+    section_info: Daar je de volgende reageerders negeert, worden hun reacties verborgen.
```
Comment (daan944) [inline plugins/talk-plugin-ignore-user/client/translations.yml]: "Daar je.." might be a bit too complex for the average Joe. Let's simplify it a bit, maybe 'Omdat je deze gebruikers negeert, worden hun reacties verborgen'. Or something like that.

File: plugins/talk-plugin-ignore-user/client/translations.yml [hunk 2]
```diff
@@ -62,17 +62,17 @@ fr:
...
+    stop_ignoring: Stop met negeren
+    ignore_user: Gebruiker negeren
+    cancel: Annuleren
     confirmation: |
-      When you ignore a user, all comments they wrote on the site will be hidden from you. You can
-      undo this later from My Profile.
+      Wanneer je een gebruiker negeert, zullen al diens reacties voor jou worden verborgen. Je kunt
```
Comment (daan944) [inline plugins/talk-plugin-ignore-user/client/translations.yml]: 'diens' - I like it, but might be too complex again. 'zijn of haar' might be better. Or 'Alle reacties van genegeerde gebruikers worden verborgen. Je kan dit later ongedaan maken vanuit Mijn Profiel. Deze gebruiker negeren?'

File: plugins/talk-plugin-moderation-actions/client/translations.yml [hunk 3]
```diff
@@ -60,16 +60,16 @@ fr:
...
-    ban_user_dialog_cancel: "Cancel"
-    ban_user_dialog_yes: "Yes. Ban user"
-    ban_user_dialog_headline: "Ban User?"
+    reject_comment: "Afkeuren"
+    approve_comment: "Goedkeuren"
+    approved_comment: "Goedgekeurd"
+    moderation_actions: "Moderatie handelingen"
+    ban_user: "Verban gebruiker"
+    ban_user_dialog_sub: "Weet je zeker dat je deze gebruiker wil verbannen?"
+    ban_user_dialog_copy: "Let op: Wanneer je deze gebruiker verbant zal zijn/haar reactie ook in de Afgekeurde reacties plaatsen."
```
Comment (daan944) [inline plugins/talk-plugin-moderation-actions/client/translations.yml]: If I read it correctly, 'plaatsen' should be 'geplaatst worden'. or 'verplaatst worden'.

File: plugins/talk-plugin-offtopic/client/translations.json [hunk 4]
```diff
@@ -21,7 +21,7 @@
   },
   "nl_NL": {
     "off_topic": "Off Topic",
-    "hide_off_topic": "Hide Off-Topic Comments"
+    "hide_off_topic": "Off-Topic reacties verbergen"
```
Comment (daan944) [inline plugins/talk-plugin-offtopic/client/translations.json]: Off-topic? Not translating this?

File: locales/nl_NL.yml [hunk 5]
```diff
@@ -62,10 +62,10 @@ nl_NL:
...
     story: 'Verhaal'
   flagged_usernames:
-    notify_approved: '{0} approved username {1}'
-    notify_rejected: '{0} rejected username {1}'
-    notify_flagged: '{0} reported username {1}'
-    notify_changed: 'user {0} changed their username to {1}' 
+    notify_approved: '{0} heeft gebruikersnaam {1} goedgekeurd'
+    notify_rejected: '{0} heeft gebruikersnaam {1} afgekeurd'
+    notify_flagged: '{0} heeft gebruikersnaam {1} gerapporteerd'
+    notify_changed: 'gebruiker {0} heeft de gebruikersnaam veranderd naar {1}'
```
[the remark this answers is not in the document]
Comment (bartee) [reply locales/nl_NL.yml]: done

File: plugins/talk-plugin-auth/client/translations.yml [hunk 6]
```diff
@@ -223,48 +223,48 @@ fr:
...
+    request_new_verify_email: "Vraag nieuwe bevestigingsemail aan"
+    verify_email: "Bedankt voor het aanmaken van een account! We hebben een email verstuurd naar het adres dat je hebt opgegeven om je account te verifiëren."
+    verify_email2: "Je account moet worden geverifiëerd voordat je kunt deelnemen in de community."
+    not_you: "Ben je dit niet?"
+    logged_in_as: "Ingelogd als"
+    facebook_sign_in: "Inloggen met Facebook"
+    facebook_sign_up: "Registreren met Facebook"
+    logout: "Uitloggen"
+    sign_in: "Inloggen"
+    sign_in_to_join: "Log in om deel te nemen"
```
[the remark this answers is not in the document]
Comment (bartee) [reply plugins/talk-plugin-auth/client/translations.yml]: Couldn't find it. Leaving it as is.

File: plugins/talk-plugin-auth/client/translations.yml [hunk 7]
```diff
@@ -223,48 +223,48 @@ fr:
...
+    username: "Gebruikersnaam"
+    already_have_an_account: "Heb je al een account?"
+    recover_password: "Wachtwoord herstellen"
+    email_in_use: "Email-adres is al in gebruik"
+    email_or_username_in_use: "Email-adres of gebruikersnaam is al in gebruik"
+    required_field: "Dit is een vereist veld"
+    passwords_dont_match: "Wachtwoorden komen niet overeen"
+    special_characters: "Gebruikersnamen kunnen alleen letters, cijfers en _ bevatten"
+    sign_in_to_comment: "Aanmelden om te reageren"
+    check_the_form: "Ongeldig formulier. Controleer je invoer"
```
[the remark this answers is not in the document]
Comment (bartee) [reply plugins/talk-plugin-auth/client/translations.yml]: I love it.

File: plugins/talk-plugin-auth/client/translations.yml [hunk 8]
```diff
@@ -223,48 +223,48 @@ fr:
...
-    required_field: "Required field"
-    save: Save
-    special_characters: "Usernames can contain letters numbers and _ only"
-    username: Username
-    write_your_username: "Edit your username"
-    your_username: "Your username appears on every comment you post."
+    check_the_form: "Ongeldig formulier. Controleer je invoer"
+    continue: "Doorgaan met dezelfde Facebook gebruikersnaam"
+    error_create: "Fout opgetreden bij het wijzigen van de gebruikersnaam"
+    fake_comment_body: "Dit is een voorbeeldreactie. Lezers kunnen hun gedachten en meningen met newsrooms delen in het reactie-gedeelte"
```
[the remark this answers is not in the document]
Comment (bartee) [reply plugins/talk-plugin-auth/client/translations.yml]: It is a default comment yeah. I can't recall having seen it on an empty comment page

File: plugins/talk-plugin-offtopic/client/translations.json (hunk 4 above)
Comment (bartee) [reply plugins/talk-plugin-offtopic/client/translations.json]: Discussed with editors. Majority wasn't in favour of translating that.

Comment (daan944) [reply plugins/talk-plugin-offtopic/client/translations.json]: OK, then please lower the T: 'Off-topic'

File: plugins/talk-plugin-auth/client/translations.yml (hunk 6 above)
[the remark this answers is not in the document]
Comment (daan944) [reply plugins/talk-plugin-auth/client/translations.yml]: OK

File: plugins/talk-plugin-auth/client/translations.yml (hunk 8 above)
[the remark this answers is not in the document]
Comment (daan944) [reply plugins/talk-plugin-auth/client/translations.yml]: Ok then it's fine it's generic.

**Pull request merged**